src/test/java/LoginLogoutTest.java

```diff
@@ -0,0 +1,51 @@
+import org.junit.jupiter.api.*;
+import org.openqa.selenium.*;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class LoginLogoutTest {
+    private  WebDriver driver;
+    private LoginPage loginPage = new LoginPage();
+    private final String username = "firstnamelastname1989";
+    private final String password = "pTKJctHpbaj@t7M";
+    private static final String EXPECTED_NAME = "Inbox — Yandex.Mail";
+
+    @BeforeEach
+    public void beforeTest () {
+        driver = SingleDriver.getSingleDriverInstance().getDriver();
+    }
+
+    @AfterEach
+    public void afterTest()  {
+     SingleDriver.getSingleDriverInstance().closeDriver();
+    }
+
+    @Test
+    @Tag("yandex_login_test")
+    @DisplayName("Login in mail test")
+    void yandexLoginTest() {
+        HomePage homePage = loginPage.loginToMail(username, password);
+
+        assertAll("HomePage",
+                () -> assertTrue(driver.getTitle().contains(EXPECTED_NAME),
+                        "Incorrect page was opened (Incorrect Title)"),
```

Comment:
It makes sense to add actual title in message

Reply:
Now it shows the current title